Resources/doc/usage.rst

```diff
@@ -24,6 +24,10 @@ In applications not using webpack add these two lines in your layout:
 If you are using webpack and Encore to package your assets you will need to use the dump command
 and export your routes to json, this command will create a json file into the ``web/js`` folder:
 
+.. note::
```

Comment:
This note is in the wrong place, as the previous paragraph expects to be followed by the code block.

Reply:
If I understand we just have to change section part of documention and put expose section on top before the dump section. What do you think ?

Reply:
Yes, that would be a nice solution.